creating sign up component  and finish styling it

mhmdwSamir opened the pull request.

mhmdwSamir committed:
creating sign up component  and finish styling it

ahmedradwan94 commented on src/app/app.component.html:
```diff
@@ -6,11 +6,7 @@
     <div class="blank-body"> 
       <app-todo-task></app-todo-task>
```
The todo components should be in routing not hard coded in `app.component.html`.
This wil solve the issue of appearing todo form and signup in the same page

mhmdwSamir replied:
ok ,
i will search for put both of them in one routing

mhmdwSamir replied:
i need more comment [ killing comments ] YA RADWAN

ahmedradwan94 commented on src/app/components/sign-up/sign-up.component.scss:
```diff
@@ -0,0 +1,80 @@
+.sign-up-section{
```
Good job for writing css classes in `kabeb case`

ahmedradwan94 reviewed (changes requested):
Create new component `TodoPageCompoment` that will be for routing `{ path: '', component: TodoPageComponent }`
And contains `app-todo-task` and `app-todo-stats`

mhmdwSamir committed:
fix routing ,
edit navbar navigation items to real names , add router link property.
add navbar routes to routing module ,

ahmedradwan94 commented on src/app/app.component.html:
```diff
@@ -1,18 +1,14 @@
     <div class="back-cover">
       <app-nav-bar></app-nav-bar>
-      <app-todo-form></app-todo-form>
+      <app-todo-form></app-todo-form> 
```
why you still render `app-todo-form` in the `app.component.html` file ? @mhmdwSamir

ahmedradwan94 commented on src/app/app-routing.module.ts:
```diff
@@ -1,7 +1,15 @@
 
-const routes: Routes = [];
+const routes: Routes = [
+  { path:  'taskStaste', component: TodoStatsComponent},
```
You should create new component as i said before `TodoPageComponent` and you can add new route like `{ path: '', component: ToDoPageComponent }` with empty string so it will be the home page.
in `todo-page.component.html` file you can add `<app-todo-stats></app-todo-stats> and <app-todo-tasks></app-todo-tasks>`.

ahmedradwan94 reviewed (commented):
I'll merge this code, and in the future tasks we will refactor a lot of it.
